### ISSUE TITLE
Incorrect indentation with multi-line string containing interpolation after a `.` character

### ISSUE THREAD

**Issue by owst** [vim-ruby/vim-ruby#311]
Using `7764bf0`, the following file:
```
def foo
  x = 1

  string = ". #{x}" \
    "xyz"

  puts string
  puts string
end
```
is indented as:
```
def foo
  x = 1

  string = ". #{x}" \
    "xyz"

    puts string
    puts string
end
```
i.e. the line(s) following the string are aligned with the second line of the string.

Strangely, if I remove _either_ the `.`, or the `#` character, the indentation is correct. The problem is also present if I replace `.` with `,`.

**Comment by AndrewRadev** [comment]
I think I've managed to fix the issue, but it's a weird one. My guess is that the code is considering `. #` as code that ends in a method call continuation and a comment, but it really shouldn't. It's supposed to check the syntax groups, so maybe it's somehow checking the wrong places. My debugging didn't really turn up with a satisfying result, but I do think we don't explicitly handle backtick continuations, so maybe that's the issue here. 

I'll likely get more information from the issues that come after this change. For now, could you pull master and let me know if it works for you?

**Comment by owst** [comment]
Yes, the problem case above is now fixed in `87b25c5`. Thanks very much for the quick turn around!
